HousingRepairsSchedulingApi.Tests/GatewaysTests/DrsAppointmentGatewayTests.cs

```diff
@@ -618,23 +620,56 @@ public async void GivenAnEndDateEarlierThanTheStartDate_WhenExecute_ThenInvalidE
 
         [Fact]
 #pragma warning disable CA1707
-        public async void GivenValidArguments_WhenExecute_ThenBookingReferenceIsReturned()
+        public async void GivenValidArguments_WhenExecute_ThenOrderIsReturned()
 #pragma warning restore CA1707
         {
             // Arrange
-            const int bookingId = 12345;
+            var bookingReference = "10003829";
+            var startDateTime = new DateTime(2022, 05, 01);
 
             drsServiceMock.Setup(x =>
-                x.CreateOrder(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<string>())
-            ).ReturnsAsync(bookingId);
+                x.SelectOrder(It.IsAny<int>(), It.IsAny<DateTime?>())
```

Comment:
As CreateOrder has been replaced by SelectOrder in the DrsAppointmentGateway, the test has been amended to reflect the change.
CreateOrder is no longer present in DrsAppointmentGateway.